api/v1alpha1/multiclusterservice_types.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2024
```

Comment:
i don't see the new resource in the `PROJECT` list, could we add it via the `kubebuilder create api` CLI call? to continue keeping track of all the resources in one (generated) place